.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: tests
+on:
+  push:
+    branches:
+      - master
+      - dev
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - dev
+      - master
+jobs:
+  test:
+    runs-on: "ubuntu-latest"
+    timeout-minutes: 40 # default is 360
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x, 14.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: npm install
```

Comment:
I feel we should run a linter as well if we have any.

Reply:
Currently, there is no linter, but we may add it on other PR.